### PR TITLE
added dynamic searchbar Glow by using onFocus and onBlur

### PR DESCRIPTION
**Pull request by Aswanth-c**
fixed issue :https://github.com/modularcloud/explorer/issues/64
Added Searchbar glows on focus

### CODE REVIEW

**Comment by Aswanth-c** [comment]
fixed the mentioned Issue https://github.com/modularcloud/explorer/issues/64
> Added Searchbar glows on focus

**Review by lzrscg — CHANGES_REQUESTED** [review]
can we do this with pure CSS? I think it makes sense to use `hover:` and `focus:`
https://tailwindcss.com/docs/hover-focus-and-other-states#pseudo-classes

what do you think?

**Comment by Aswanth-c** [comment]
> can we do this with pure CSS? I think it makes sense to use `hover:` and `focus:` https://tailwindcss.com/docs/hover-focus-and-other-states#pseudo-classes
> 
> what do you think?

Yep, I tried that, but it doesn't work because the input element is only 1/3, and we want the glow on the whole div element. 

eg, if I try to use  focus in the input area 
image

Another thing I want to mention is I created a wrapper div and styled it because it's better to have a wrapper div if we are using gradient color for  search barglow

File: apps/web/app/[network]/[type]/(standard)/[query]/[[...viewPath]]/(components)/search.tsx [hunk 1]
```diff
@@ -19,76 +19,90 @@ export const Search = ({ optionGroups, defaultValue }: Props) => {
...
-              type="text"
-              placeholder="Go to hash, height, or address"
-            />
-            <button
-              onClick={() => {
-                if (searchInput.current)
-                  router.push(`/${option}/search/${searchInput.current.value}`);
-              }}
-              className="px-3 flex items-center"
+        <div className={searchBarGlow ? "p-[1px] rounded-lg bg-[#9DAAE2]" : ""}>
```
Comment (lzrscg) [inline apps/web/app/[network]/[type]/(standard)/[query]/[[...viewPath]]/(components)/search.tsx:30]: oh good point, then just add the pseudo class here and user `hover:`

`focus:` won't work but its not necessary

Comment (Aswanth-c) [reply apps/web/app/[network]/[type]/(standard)/[query]/[[...viewPath]]/(components)/search.tsx]: Did you mean adding the glow on hover also? I have added that on the latest commit

**Comment by lzrscg** [comment]
this needs to be a super simple CSS only change

when the search bar (not text input box) is hovered, it glows (using the CSS pseudoclass)

**Comment by lzrscg** [comment]
closing for now

**Pull request closed without merging**